src/NeuralNet/Initializers/Normal/Exceptions/InvalidStandardDeviationException.php

```diff
@@ -0,0 +1,14 @@
+<?php
```

Comment:
I have similar thoughts about this exception class as I do with the invalid fan in/fan out ones. I'm not seeing any extra logic and I don't foresee that users will need to catch this exception specifically.